h2o-py/tests/testdir_parser/pyunit_PUBDEV_5705_drop_columns_parser_gz_large.py

```diff
@@ -0,0 +1,90 @@
+from __future__ import print_function
+import sys
+
+sys.path.insert(1, "../../")
+import h2o
+from tests import pyunit_utils
+import random
+
+def import_zip_skipped_columns():
```

Comment:
Why does the filename of this test has "large" postfix ?

Reply:
large is added to tests that takes longer to complete.  They will be put into different directories and give more time to finish